src/components/SigningOfficialDaaAgreementWrapper.js

```diff
@@ -2,21 +2,29 @@ import { h, h2, a, p, div, span, button } from 'react-hyperscript-helpers';
 import { Notifications } from '../libs/utils';
 import BroadLibraryCardAgreementLink from '../assets/Library_Card_Agreement_2021.pdf';
 import NIHLibraryCardAgreementLink from '../assets/NIH_Library_Card_Agreement_11_17_22_version.pdf';
+import DataSubmitterAgreementLink from '../assets/Data_Registrant_Agreement_7.2.24.22.pdf';
 
 import Acknowledgments, {acceptAcknowledgments, hasSOAcceptedDAAs} from '../libs/acknowledgements';
 import { useEffect, useState } from 'react';
 import { spinnerService } from '../libs/spinner-service';
-import { isNil } from 'lodash';
+import { isNil, isNull } from 'lodash';
 import { Styles } from '../libs/theme';
 
 export const SigningOfficialDaaAgreementWrapper = (props) => {
   const {
     onAccept,
-    children
+    children,
   } = props;
 
+  const {
+    props: {
+      dataSubmitter,
+    },
+  } = children;
+
```

Comment:
I wonder if there's a better way to do this, the repeated 'props' element causes issues.